spec/features/user/user_sees_github_repos_on_dashboard_spec.rb

```diff
@@ -25,20 +25,21 @@
       end
     end
 
-    it 'does not display if user missing a gd token' do
+    it "I do not see a GitHub section if I'm missing a GitHub token" do
```

Comment:
👍

Reply:
Good refactor to be more specific about what is being tested